src/utils/__test__/mapsheet.test.ts

```diff
@@ -19,6 +19,20 @@ describe('MapSheets', () => {
     });
   });
 
+  it('should generate 1:50k name', () => {
+    assert.deepEqual(MapSheet.extract('CK08_50000_0101.tiff'), {
+      mapSheet: 'CK08',
+      gridSize: 50000,
```

Comment:
Nit: ECMAScript 2021 supports [numeric separators](https://v8.dev/features/numeric-separators), so that big numbers can be written like `50_000` for clarity. Also, we use such separators elsewhere in this repo, so we should probably do the same here for consistency.